source/agora/test/MultiRoundConsensus.d

```diff
@@ -138,14 +138,14 @@ unittest
     auto validator = network.clients[0];
 
     // Make two of three validators stop responding
-    nodes[1].ctrl.sleep(10.seconds, true);
-    nodes[2].ctrl.sleep(10.seconds, true);
+    nodes[1].ctrl.sleep(conf.timeout, true);
+    nodes[2].ctrl.sleep(conf.timeout, true);
```

Comment:
Do you know if we need the 10s timeout to begin with ?

Reply:
I assume it was set to match the node timeout set in the TestConf.
I think we can use the timeout of 5 secs in the TestConf for this test and then we will only sleep for 5 secs here and the final expect block should be within 10 secs. The number of rounds will then only be >=2 though. I will try it out.